packages/docs/src/styles/components/_UsaBanner.scss

```diff
@@ -0,0 +1,171 @@
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    font-family: Source Sans Pro Web,Helvetica Neue,Helvetica,Roboto,Arial,sans-serif;
+    font-size: 1.06471rem;
+    line-height: 1.52155;
+  }
+  .usa-banner__header {
+    padding-bottom: .25rem;
+    padding-top: .25rem;
+    font-size: .79853rem;
+    font-weight: 400;
+    min-height: 0;
+    position: relative;
+}
+.usa-banner__inner {
+    margin-left: auto;
+    margin-right: auto;
+    padding-left: 1rem;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-wrap: wrap;
+    flex-wrap: wrap;
+    -webkit-box-align: start;
+    -ms-flex-align: start;
+    align-items: flex-start;
+    padding-right: 0;
+}
+[class*=grid-col] {
+    position: relative;
+    width: 100%;
+    -webkit-box-sizing: border-box;
+    box-sizing: border-box;
+}
+.usa-banner__header-flag {
+    float: left;
+    margin-right: .5rem;
+    width: 1rem;
+}
+.usa-banner__header-text {
+    margin-bottom: 0;
+    margin-top: 0;
+    font-size: .79853rem;
+    line-height: 1.12707;
+}
+.usa-banner__header-action {
+    color: #005ea2;
+    line-height: 1.12707;
+    margin-bottom: 0;
+    text-decoration: underline;
+    font-family: Helvetica;
+    &::before {
+      content: none;
+    }
+}
+.usa-banner__button {
+    position: static;
+    bottom: auto;
+    left: auto;
+    right: auto;
+    top: auto;
+    display: inline;
+    margin-left: .5rem;
+    position: relative;
+    -moz-osx-font-smoothing: auto;
+-webkit-font-smoothing: subpixel-antialiased;
+background-color: transparent;
+border: 0;
+border-radius: 0;
+-webkit-box-shadow: none;
+box-shadow: none;
+font-weight: 400;
+margin: 0;
+text-align: left;
+left: 0;
+bottom: 0;
+top: 0;
+color: #005ea2;
+text-decoration: underline;
+vertical-align: baseline;
+display: block;
+font-size: .79853rem;
+height: auto;
+line-height: 1.12707;
+padding: 0;
+text-decoration: underline;
+width: auto;
+margin-left: .5rem;
+}
+.usa-banner__content {
+    margin-right: auto;
+    max-width: 64rem;
+    background-color: transparent;
+    font-size: .99816rem;
+    overflow: hidden;
+    padding: 1rem;
+    width: 100%;
+}
+.usa-banner__icon {
+    width: 2.5rem;
+}
+.usa-media-block__img {
+    float: left;
+    margin-right: .5rem;
+}
+.usa-media-block__body {
+    overflow: hidden;
+}
+.usa-banner__content p:first-child {
+    margin: 0;
+}
+.grid-row {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-wrap: wrap;
+    flex-wrap: wrap;
+}
+.grid-col-fill {
+    -webkit-box-flex: 1;
+    -ms-flex: 1 0 0px;
+    flex: 1 0 0;
+    width: auto;
+    max-width: 100%;
+}
+.grid-col-auto {
+    -webkit-box-flex: 0;
+    -ms-flex: 0 0 auto;
+    flex: 0 0 auto;
+    width: auto;
+max-width: 100%;
+}
+.tablet\:grid-col-1, .tablet\:grid-col-auto {
+    -webkit-box-flex: 0;
+    -ms-flex: 0 0 auto;
+    flex: 0 0 auto;
+}
+.tablet\:grid-col-6 {
+    width: 50%;
+}
+.usa-banner__header--expanded {
```

Comment:
I think there should be some css here as well to make the arrow flip to point up.